src/test/java/utils/TestModelsFactory.java

```diff
@@ -107,7 +107,7 @@ public static Driver createTestDriver3(Address address){
         driver3.setAvailable(true);
         driver3.setDeleted(false);
         return driver3;
-    }
+    }*/
```

Comment:
If this method is not used anywhere - delete it. IMO there is no logic in this method that is worth keeping.